### PR TITLE
Improve printing and errors

### PR DESCRIPTION
**Pull request by ChrisTerBeke**
## What
Use helper functions for commonly used printing and error handling.

## Why
* Easier to maintain the codebase.
* Reuse functions like color-coded versions.

### CODE REVIEW

File: pkg/helpers/helper.go [hunk 1]
```diff
@@ -7,41 +7,77 @@ import (
...
+}
+
+// ExitWithError prints an error message and exits with status code 1
+func ExitWithError(message string, err error) {
+	fmt.Printf("%s %s: %s\n", color.HiRedString("error:"), message, err)
+	os.Exit(1)
+}
+
+// ErorWithHelp returns an error with a help message
+func ErorWithHelp(help string) error {
```
Comment (bschaatsbergen) [inline pkg/helpers/helper.go:34]: Small typo: Error > Eror

**Review by bschaatsbergen — COMMENTED** [review]
Nice fixes! Small typo here

File: pkg/helpers/helper.go (hunk 1 above)
Comment (bschaatsbergen) [reply pkg/helpers/helper.go]: Fixed in: [c7cc9b2](https://github.com/tfversion/tfversion/pull/15/commits/c7cc9b26c65f80649b04487f99108b1bacf6861a)

**Review by bschaatsbergen — APPROVED** [review]
LGTM 🚀

File: pkg/helpers/helper.go (hunk 1 above)
Comment (ChrisTerBeke) [reply pkg/helpers/helper.go]: lol derp typo

**Pull request merged**